packages/opentelemetry-tracing/test/Tracer.test.ts

```diff
@@ -220,4 +221,37 @@ describe('Tracer', () => {
     assert.strictEqual(context.traceFlags, TraceFlags.NONE);
     span.end();
   });
+
+  it('should start an active span', () => {
+    const tracer = new Tracer(
+      { name: 'default', version: '0.0.1' },
+      { sampler: new TestSampler() },
+      tracerProvider
+    );
+
+    assert.strictEqual( tracer.startActiveSpan('my-span', span => {
+      try {
+        return 1
+      } finally {
+        span.end();
+      }
+    }), 1);
+
+    assert.strictEqual( tracer.startActiveSpan('my-span', {}, span => {
```

Comment:
each of these cases should be separate `it()` test cases.

Reply:
Please assert that the span options are actually used. This can be done by using sinon to mock startSpan on the tracer.

Reply:
I've used sinon spy for this, ptal